src/components/SkillDetails.tsx

```diff
@@ -0,0 +1,57 @@
+import { useQuery } from "@apollo/client/react";
+import React, { useContext } from "react";
+import { SkillDetailsQuery } from "../generated/graphql";
+import { GET_SKILL_DETAILS } from "../graphql/queries/skills";
+import { Skill } from "../utils/types";
+import { i18nContext } from "../utils/i18nContext";
+import { useAuth0 } from "@auth0/auth0-react";
+
+const SkillDetails = ({ skill }: { skill: Skill }) => {
+  const { user } = useAuth0();
+  const { t } = useContext(i18nContext);
+  const {
+    data: skillDetails,
+    error,
+    loading,
+  } = useQuery<SkillDetailsQuery>(GET_SKILL_DETAILS, {
+    variables: { skillId: skill.id || skill.skillId, email: user.email },
+    fetchPolicy: "network-only",
+  });
+
+  return !loading ? (
+    !error || !skillDetails ? (
```

Comment:
```suggestion
    !error || skillDetails ? (
```

Do you mean this ? Otherwise there will be an error if loading is done and there is no data.

Reply:
oups, well seen !